utils.py

```diff
@@ -267,21 +268,43 @@ def get_random_choice(alist):
     return alist[index]
 
 
+def convert_png_data_url_to_binary(content):
```

Comment:
Why not just call it image_data_url rather than "content"? Then you don't need the clarification in the docstring.

Reply:
Done.